src/components/Identity/injectAwaitIdentityCookie.js

```diff
@@ -24,18 +24,16 @@ export default ({ doesIdentityCookieExist, orgId }) => {
         } else {
           // This logic assumes that the code setting the cookie is working as expected and that
           // the cookie was missing from the response.
-          const noIdentityCookieError = new Error(
-            `An identity was not set properly. Please verify that the org ID ${orgId} configured in Alloy matches the org ID specified in the edge configuration.`
+          logger.warn(
```

Comment:
I like that we're providing multiple reasons for this error to occur. I would love to get us to a place where we don't check for this cookie at all. There are multiple Identity related conversations happening soon that might help us get there.

Reply:
I think this error is masking the true error that is occurring. I'm hoping that if we release this, we'll get fewer reports of this error, because the cause of the problem will be apparent.

Reply:
Do we have any visibility/logging into the "true error"

Reply:
The true error will be the one that rejects the promise so it should be visible in the console. I suspect that it will point to customer's code and we won't hear about it.